### PR TITLE
chore: update documentation site

### PR DESCRIPTION
**Pull request by awentzel**
# Description
1. Update status page to reflect changes made to pipelines recently
2. Update TypeDoc naming to be case sensitive when used in documentation
3. Rebuild API documentation

## Motivation & context
Keeping up with latest DevOps and code changes
closed #1509 #1536 #1394 #1497

## Issue type checklist

- [x] **Chore**: A change that does not impact distributed packages.
- [ ] **Bug fix**: A change that fixes an issue, link to the issue above.
- [ ] **New feature**: A change that adds functionality.

**Is this a breaking change?**
- [ ] This change causes current functionality to break.

## Process & policy checklist

- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [x] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast-dna/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://microsoft.github.io/fast-dna/docs/en/contributing/standards) for this project.

### CODE REVIEW

**Comment by janechu** [comment]
I would add to @chrisdholt comment that this should be automated as well, we don't want to have to create PRs for generated files that should be added when we update any README.

**Comment by awentzel** [comment]
I've addressed all concerns @janechu @chrisdholt please review

File: website/i18n/en.json [hunk 1]
```diff
@@ -37,117 +37,3698 @@
         "title": "Writing style guide",
         "sidebar_label": "Writing style guide"
       },
+      "en/packages/fast-animation/api/classes/_animate_index_.animate": {
```
Comment (janechu) [inline website/i18n/en.json]: is this file being auto-generated? maybe we should be ignoring it.

File: website/sidebars.json [hunk 2]
```diff
@@ -37,6 +37,7 @@
       "en/packages/fast-navigation-generator-react/index",
       "en/packages/fast-permutator/index",
       "en/packages/fast-sketch-library/index",
+      "en/packages/fast-tooling-react/index",
```
Comment (janechu) [inline website/sidebars.json]: same question with this file, is it being auto-generated?

Comment (awentzel) [reply website/sidebars.json]: this file is not autogenerated.

**Pull request merged**